### PR TITLE
[Wait for #1427] [ LSTM ] Add drop out mask

### PR DESCRIPTION
**Pull request by jijoongmoon**
In this commit,
 . drop out mask is added.
 . calculation of drop out is implemented.

**Self evaluation:**
1. Build test:	 [X]Passed [ ]Failed [ ]Skipped
2. Run test:	 [X]Passed [ ]Failed [ ]Skipped

Signed-off-by: jijoong.moon <jijoong.moon@samsung.com>

### CODE REVIEW

File: nntrainer/layers/lstm.h [hunk 1]
```diff
@@ -139,6 +140,8 @@ class LSTMLayer : public LayerImpl {
    */
   float dropout_rate;
 
+  float epsilon;
```
Comment (kparichay) [inline nntrainer/layers/lstm.h:143]: Please add description.

File: nntrainer/layers/lstm.cpp [hunk 2]
```diff
@@ -52,6 +60,11 @@ void LSTMLayer::finalize(InitLayerContext &context) {
   output_dim = input_dim;
   output_dim.width(unit);
 
+  if (dropout_rate > epsilon) {
+    wt_idx[LSTMParams::dropout_mask] = context.requestTensor(
+      output_dim, "LSTM:dropout_mask", true, ITERATION_LIFESPAN);
```
Comment (kparichay) [inline nntrainer/layers/lstm.cpp:65]: `true` ->  `false`
https://github.com/nnstreamer/nntrainer/pull/1427#discussion_r676238140

Comment (jijoongmoon) [reply nntrainer/layers/lstm.cpp:65]: I will fix it.

**Review by kparichay — APPROVED** [review]
LGTM

**Review by zhoonit — APPROVED** [review]
LGTM

**Pull request merged**